tests/test_wsi_registration.py

```diff
@@ -20,24 +21,27 @@
 
 def test_extract_features(dfbr_features):
     """Test for CNN based feature extraction function."""
-    df = DFBRegister()
```

Comment:
This could probably have a more descriptive name

Reply:
Added a description for dfbr in comments.